pkg/storaged/stratis/stopped-pool.jsx

```diff
@@ -34,18 +34,57 @@ import {
 import { dialog_open, PassInput } from "../dialog.jsx";
 import { std_reply, with_stored_passphrase } from "./utils.jsx";
 
+import * as python from "python.js";
+import stratis3_start_pool_py from "./stratis3-start-pool.py";
+
 const _ = cockpit.gettext;
 
+async function stratis3_r8_start_pool(uuid, passphrase) {
+    if (passphrase) {
+        return await python.spawn(stratis3_start_pool_py, [uuid, "passphrase"], { superuser: "require" })
+                .input(passphrase);
+    } else {
+        return await python.spawn(stratis3_start_pool_py, [uuid, "clevis"], { superuser: "require" });
+    }
+}
+
 function start_pool(uuid, show_devs) {
     const devs = client.stratis_manager.StoppedPools[uuid].devs.v.map(d => d.devnode).sort();
     const key_desc = client.stratis_stopped_pool_key_description[uuid];
     const clevis_info = client.stratis_stopped_pool_clevis_info[uuid];
 
+    // HACK - if this is a V2 encrypted pool, it needs to be started
+    //        with the r8 StartPool method.
+
+    const r8_stopped_pool = client.stratis_manager_r8?.StoppedPools?.[uuid];
+    const v2_encrypted = (
+        r8_stopped_pool &&
+            r8_stopped_pool.metadata_version.v[0] &&
+            r8_stopped_pool.metadata_version.v[1] == 2 &&
+            r8_stopped_pool.features.v[0] &&
+            r8_stopped_pool.features.v[1].encryption);
```

Comment:
These 4 added lines are not executed by any test.

Reply:
V2 stuff is executed on Fedora 42.